.storybook/pages/cds-core.stories.mdx

```diff
@@ -0,0 +1,8 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="cds-core" />
```

Comment:
Shouldn't this go under "Theming" section, or at least be titled "Theme", as right now it just hangs under "Home" and the name "cds-core" does not bring any idea what type of information is there inside it.

Reply:
Fixed by @valentin-mladenov with this commit - https://github.com/vmware-clarity/ng-clarity/pull/1100/commits/cb131dca99d705342af88d322b1ecd8fdd9d195b